README.md

```diff
@@ -258,146 +273,40 @@ In general, you should never need to commit a submodule update to `terraform-web
 
-In this case, the failing job usually doesn't mean the actual build or deploy failed, and instead means that the link-checking or cache-warming scripts found a broken link and exited with a non-zero status code.
+All of these jobs are configured in `./circleci/config.yml`, in this repo and in hashicorp/terraform.
 
-### Step 1a: Identify the First Bad Build
+### Global Link Check
 
-If you noticed the failing builds early enough, you might be able to see the exact point where they flipped from green to red. If so, look at the commit associated with the first bad build!
+We run a global link check for the whole site after every deploy.
 
-Knowing the commit can save you a lot of time in step 4, when you're trying to figure out which files include the broken links.
+- **Where:** The job reports its status in the `#proj-terraform-docs` channel in Hashicorp's Slack.
+- **What:** This job only checks internal links within terraform.io, not external links to the rest of the web. (We deploy a _lot,_ and we don't want to be a nuisance on someone else's servers.)
+- **Who:** The Terraform Education team is ultimately responsible for dealing with any broken links this turns up, but anyone in the channel is welcome to fix something if they see it first!
+- **How:** We're using [filiph/linkcheck](https://github.com/filiph/linkcheck/) for this. In addition to checking links, this also warms up the Fastly cache for the site.
 
-If the build stayed red long enough for additional broken links to creep in, this will only help find the first batch. Thus, it's easier to deal with broken links if you catch them fast.
+### PR Link Check
 
-### Step 2: Find the Broken URL(s) in the Build Log
+We run a targeted link check for docs PRs, in this repo and in hashicorp/terraform.
 
-If you see a red build (on a PR or in the chatroom), click through to the build log. Once it's loaded, Cmd-F and search for the text "broken link". There might be more than one message, so make sure to search repeatedly (Cmd-G) to catch them all.
+- **Where:** It shows up as a GitHub PR check. It only runs for PRs from people in the HashiCorp GitHub organization (which should be fine, since we're the most likely to change a bunch of links at once.)
+- **What:** This job only checks links in the _content area_ (not navs/headers) of _pages that were changed in the current PR._ It checks both internal and external links.
+- **Who:** If this job is red in your PR, please fix your broken links before merging! Alternately, if it throws a false-positive and complains about a link that is actually fine, make sure to explain that before merging.
+- **How:** This is a custom-built Ruby script, because we weren't able to find an off-the-shelf link checker that met our requirements (i.e. don't complain about problems that have nothing to do with this PR).
```

Comment:
Should we link to the script from here?

Reply:
```suggestion
- **How:** This is a custom Ruby script, because we weren't able to find an off-the-shelf link checker that met our requirements (i.e. don't complain about problems that have nothing to do with this PR). ([content/scripts/check-pr-links.rb](./content/scripts/check-pr-links.rb))
```